src/Samples/TestEmbedded/UnitTestEmbedded/UnitTestEmbedded.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.0;net5.0;net6.0</TargetFrameworks>
```

Comment:
Wondering why we're using netcoreapp2.0 here?

Reply:
This is to test the netstandard projections, we do the same thing in our existing UnitTest.csproj